waku/v2/node/rest/debug_api.yaml

```diff
@@ -0,0 +1,34 @@
+openapi: 3.1.0
```

Comment:
To me it makes sense to have the `yaml` files separate. Perhaps a `/node/rest/yaml/` or `/node/rest/openapi/` subdirectory. WDYT?

Reply:
In my opinion, these YAML files should be collocated with the different request handlers—something like this:

```
rest/
├─ debug/
│  ├─ debug.yaml
│  ├─ debug_api_handlers.nim
│  ├─ debug_api_models.nim
├─ store/
│  ├─ store.yaml
│  ├─ store_api_handlers.nim
│  ├─ store_api_models.nim
├─ ...
```

The reason behind this is the [_single responsibility principle_](https://www.oreilly.com/library/view/97-things-every/9780596809515/ch76.html):

> Gather together those things that change for the same reason, and separate those things that change for different reasons.

This cannot be appreciated yet because I am still evaluating all the pros and cons of that approach in Nim and nim-presto. From my experience, collocating everything in that "structured" way boosts maintainability (e.g. removing a resource, or namespace in our case, is as simple as removing one folder).

If you see any other reason (that probably I am not taking into account) to segregate the files, we can have a call and discuss 😁